libqtile/images.py

```diff
@@ -8,6 +8,11 @@
 # The above copyright notice and this permission notice shall be included in
 # all copies or substantial portions of the Software.
 
+import io
+import os
+import re
+from collections import OrderedDict, defaultdict, namedtuple
+
```

Comment:
Hmm, this doesn't seem right. I think we should concatenate both comments section to the imports are not inserted between the two.